src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/Component_ComplexContentInAttribute/TestComponent.codegen.cs

```diff
@@ -20,15 +20,15 @@ public partial class TestComponent : global::Microsoft.AspNetCore.Components.Com
         #pragma warning disable 1998
         protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder __builder)
         {
-            __o = global::Microsoft.AspNetCore.Components.CompilerServices.RuntimeHelpers.TypeCheck<global::System.String>(
```

Comment:
Why did this lose the typecheck? Just because the nodes in the tree changed?

Reply:
Yes. Changed to emit mixed content same as normal content so the type check is there.